src/WebApp/Extensions/Extensions.cs

```diff
@@ -103,6 +103,7 @@ private static void AddAIServices(this IHostApplicationBuilder builder)
 
         if (!string.IsNullOrWhiteSpace(builder.Configuration.GetConnectionString("openai")) && !string.IsNullOrWhiteSpace(deploymentName))
         {
+            builder.Services.AddKernel();
```

Comment:
Looking at my PR (which removed it, probably by mistake)_it was then registering the services on the returned builder `var kernelBuilder = builder.Services.AddKernel();`. I check the source and it's the same as `builder.Service` so this is fine.